code/modules/mob/living/simple_animal/friendly/raven.dm

```diff
@@ -0,0 +1,49 @@
+///the perfectly normal raven.
+/mob/living/simple_animal/pet/raven
+	name = "raven"
+	desc = "Quoth the raven nevermore! Just now in space!"
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "raven"
+	icon_living = "raven"
+	icon_dead = "raven_dead"
+	icon_resting = "raven"
+	speak_emote = list("caws")
+	health = 7
+	maxHealth = 7
+	attacktext = "bites"
+	melee_damage_lower = 2
+	melee_damage_upper = 2
+	response_help  = "pets"
+	response_disarm = "shoos"
+	response_harm   = "kicks"
+	ventcrawler = VENTCRAWLER_ALWAYS
+	density = FALSE
+	pass_flags = PASSTABLE | PASSMOB
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat = 2)
+	gold_core_spawnable = FRIENDLY_SPAWN
+	turns_per_move = 5
+	///This varible allows you to set a cooldown on an ability, for an example usage reference the raven.dm, last line.
+	var/caw_cooldown
+
+
+/mob/living/simple_animal/pet/raven/npc_safe(mob/user)//Makes the raven a playable via the ghost menu.
+	return TRUE
+//The raven category is established here, as well as the fly and take off button
+/mob/living/simple_animal/pet/raven/verb/perch()
+	set name = "Fly/Take off!"
+	set category = "Raven"
```

Comment:
This should probably be in an existing category as new ones will cause lag.

Reply:
I am not entirely how to resolve this one, is there a specific file I should define it in?

Reply:
Also thank you for taking time to review my code, I thought I was on my own now.

Reply:
Yes, one of the existing verb categories, I'd probably go with `IC`

Reply:
![image](https://user-images.githubusercontent.com/28615254/196298711-afe4fa27-ab90-4c0a-a6c5-866ea0856664.png)